examples-not-proccessed/.gitrepo

```diff
@@ -0,0 +1,11 @@
+; DO NOT EDIT (unless you know what you are doing)
```

Comment:
Is the example directory still a subrepo?

Reply:
No, this coupling should be removed.